apps/kg/models/pytorch/tensor_models.py

```diff
@@ -23,7 +23,7 @@
 logsigmoid = functional.logsigmoid
 
 def get_device(args):
-    return th.device('cpu') if args.gpu < 0 else th.device('cuda:' + str(args.gpu))
+    return th.device('cpu') if args.gpu[0] < 0 else th.device('cuda:' + str(args.gpu[0]))
```

Comment:
this function seems to not work correctly since we have multiple devices now.

Reply:
For multiple gpu, mix_cpu_gpu should be used and get_device is not used at that time.